lib/react.dart

```diff
@@ -122,7 +120,7 @@ abstract class Component {
   /// [doesn't work for overriding fields](https://github.com/dart-lang/sdk/issues/27452).
   ///
   /// TODO: Switch back to a plain field once this issue is fixed.
-  Map _props;
+  late Map _props;
```

Comment:
IMO, the use of `late` here deserves a comment - even if it isn't part of the documentation comment.

Reply:
Addressed in https://github.com/Workiva/react-dart/pull/373/commits/ee90271ade553281b32779002dbb03325e20db4d